frontend/utils/questionnaires.js

```diff
@@ -201,11 +201,16 @@ export function mapQuestionnaireTypes(qTypes) {
                             min++
                         }
                     } else if(question.type === "slider") {
+                        let showTickLabels = true
+                        if (typeof question.showTickLabels !== 'undefined') {
+                            showTickLabels = question.showTickLabels
+                        }
                         question.config = {
                             min: question.min,
                             max: question.max,
                             minLabel: question.minLabel,
-                            maxLabel: question.maxLabel
+                            maxLabel: question.maxLabel,
+                            showTickLabels
```

Comment:
I have simplified to a one-liner `showTickLabels: question.showTickLabels ?? true`, the `??` operator is necessary because we need to check for undefined also, as the `!!` prefix doesn't work if the variable is not defined 😅

Reply:
That's weird because `!!` is a shortcut for `Boolean()` and `Boolean(undefined)` should return true, but I will go with your suggestion XD

Reply:
I actually agree with you, but when I tested it, it didn't work, which is strange but.... yeah. XD

Reply:
*false